src/app/gateway/gateway-edit/gateway-edit.component.ts

```diff
@@ -72,36 +72,35 @@ export class GatewayEditComponent implements OnInit, OnDestroy {
       latitude: this.gateway.location.latitude,
       draggable: true,
       useGeolocation: !this.editMode,
-      editMode: this.editMode
+      editMode: this.editMode,
     };
   }
 
   createGateway(): void {
```

Comment:
I'm not sure if this should be fixed, but if we create a gateway with an existing ID or existing name, we just get "Bad Request" as an error, instead of something more useful, like you need an unique ID or unique name :-)

Reply:
OBS: When change to chirpstack V4 there will be a different handling of the post and put request since GRPC is used. Right now, the wrongly exceptions is thrown based on the http response, so a fix should not be made right now, but we should be aware of it when the chirpstack is merged later on.

Reply:
Ah, this happens because the call to chirpstack fails and the error is not forwarded/checked for error fields